### PR TITLE
Additional Player UI fixes

### PR DESCRIPTION
**Pull request by yschimke**
#### WHAT

image

Also test on different sizes.

#### WHY

Fix for smaller watch displays

#### Checklist :clipboard:
- [x] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [x] Update metalava's signature text files

### CODE REVIEW

File: audio-ui/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -15,8 +15,15 @@
...
 
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="com.google.android.horologist.audio.ui">
 
     <uses-feature android:name="android.hardware.type.watch" />
     <uses-permission android:name="android.permission.VIBRATE" />
+
+    <!-- Tiles is API 26, but if we don't stop this here, then we can't run the sample app
+    on older devices (25).  We also don't want this to bleed into other modules via
```
Comment (yschimke) [inline audio-ui/src/main/AndroidManifest.xml:25]: accident

File: audio-ui/src/androidTest/java/com/google/android/horologist/audio/ui/VolumeScreenTest.kt [hunk 2]
```diff
@@ -59,12 +64,13 @@ class VolumeScreenTest {
...
-                        .fillMaxSize().focusRequester(focusRequester),
-                    model
-                )
+            val density = LocalDensity.current
+            val config = LocalConfiguration.current
+
+            Box(modifier = Modifier.fillMaxSize(), contentAlignment = Alignment.Center) {
+                Column {
+                    Text("densityDpi" + config.densityDpi)
+                }
```
Comment (luizgrp) [inline audio-ui/src/androidTest/java/com/google/android/horologist/audio/ui/VolumeScreenTest.kt:73]: question: intend to merge this or was it used to debug something else?

**Review by luizgrp — APPROVED** [review]

File: audio-ui/src/androidTest/java/com/google/android/horologist/audio/ui/VolumeScreenTest.kt (hunk 2 above)
Comment (yschimke) [reply audio-ui/src/androidTest/java/com/google/android/horologist/audio/ui/VolumeScreenTest.kt:73]: Ooops

File: media-ui/src/main/java/com/google/android/horologist/media/ui/components/TextMediaDisplay.kt [hunk 3]
```diff
@@ -46,11 +43,9 @@ public fun TextMediaDisplay(
             modifier = Modifier.fillMaxWidth(0.7f),
             color = MaterialTheme.colors.onBackground,
             textAlign = TextAlign.Center,
-            overflow = TextOverflow.Ellipsis,
```
[the remark this answers is not in the document]
Comment (yschimke) [reply media-ui/src/main/java/com/google/android/horologist/media/ui/components/TextMediaDisplay.kt:49]: Yeah, and before we have that it does appear to fix the truncation at the start.

**Pull request merged**